src/App.tsx

```diff
@@ -1,24 +1,156 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { TodoHeader } from './components/TodoHeader';
+import { client } from './utils/fetchClient';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { TodoFooter } from './components/TodoFooter';
+import { Notification } from './components/Notification';
+import { FilterBy } from './types/FilterBy';
+import { Errors } from './types/Errors';
 
-const USER_ID = 0;
+import * as utils from './utils/utils';
+
```

Comment:
check path to fetchClient
in folder utils no any foldet or file with name utils.
or maybe you forget to push all files.
![Screenshot from 2023-08-06 16-09-11](https://github.com/mate-academy/react_todo-app-add-and-delete/assets/33029864/41ae7bd8-ec7c-4f68-9b9f-b38f1582024e)
![Screenshot from 2023-08-06 16-10-52](https://github.com/mate-academy/react_todo-app-add-and-delete/assets/33029864/0377f736-d823-4db5-bb63-9a1cdea45f40)